malio-gwt2-tests/src/main/java/com/github/nalukit/malio/test/model/validateitem01/Address.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright © 2023 Frank Hossfeld, Philipp Kohl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.nalukit.malio.test.model.validateitem01;
```

Comment:
does this issue refer more to subvalidator than to validateitem, which is a new feature request?

Reply:
Not really, it shows, that arrays are now ignored. They will get more value with the upcoming PR.